h2o-genmodel/src/main/java/hex/genmodel/attributes/parameters/KeyValue.java

```diff
@@ -22,4 +24,34 @@ public double getValue() {
     public String toString() {
         return "{Key: " + key + ", Value: " + value + "}";
     }
+    
+    public static class AscComparator implements Comparator<KeyValue> {
```

Comment:
I think if you have a specific functionality, it would be better to call the outer class something else, eg. FeatureContribution.

Reply:
This class is used for MojoDeserialization, Variable importance sorting and Contributions sorting. I think there nothing special, just pair of string-double values and couple of comparators to sort by values

Reply:
oh, sorry, didn't notice you are actually reusing something that is used for attribute parsing

In that case, using this code for algo execution is not advised. I suggest creating a new class that addresses your specific functionality, eg. FeatureContributions.

At least your comparators definitely don't belong here.

Reply:
All right, thanks, I'll do it

Reply:
thanks, the classes meant for parsing of info from bundle modeldetails.json should be independent of the regular scoring logic 

maybe it would be better if it was a separate module but that would make build more complicated - something to thing about

Reply:
KeyValue class is untouched